lib/src/main/java/com/fabernovel/statefullayout/State.kt

```diff
@@ -112,10 +112,13 @@ class State : FrameLayout {
         }
     }
 
-    internal fun show(fallbackTransitionProvider: StateTransitionProvider? = null) {
+    internal fun show(
+        fallbackTransitionProvider: StateTransitionProvider? = null,
+        playTransition: Boolean = true
+    ) {
         currentTransition?.cancel()
         val transition = enterTransition ?: fallbackTransitionProvider?.get()
-        if (transition == null) {
+        if (transition == null || !playTransition) {
```

Comment:
You explicitly say "play the transition !" but in some cases no transition will happen, it might be confusing.

What do you think of creating a global parameter in the StatefulLayout, to enable or disable any Animation ?
If you're not willing to do this, playTransition could be renamed

Reply:
I think, I should both rename and add an attribute allowing to disable animation